### ISSUE TITLE
Support for repeated commands 

### ISSUE THREAD

**Issue by meyercm** [BlueM/cliclick#46]
I know that `cliclick` can work from a commands file, or that I could write a bash loop; however, I'd love to see syntax like (inspired by xdotool):

`> cliclick --repeat 500 -w 20 c:.`  => generate 500 click events at the current mouse position, one every 20ms

`cliclick` has a nice command chaining syntax, so maybe a better approach is to extend the algebra you've already written:

`> cliclick -w 20 c:.*500` (same 500 clicks, once every 20ms as before)

allowing paren would enable more complicated logic:

`> cliclick (c:.*5 m:+100,+100 w:20)*5` (5 clicks, move, wait, repeat all 4 more times)

**Comment by meyercm** [comment]
I think your proposed syntax looks fine.

Frankly, my use case would be for autoclicking scenarios, and wouldn't need any grouping logic- I proposed it as a natural extension to the repeat feature.

**Comment by BlueM** [comment]
No, not implemented – that’s the reason why this issue is still open.

**Comment by paxperscientiam** [comment]
In the mean time, one could do something like this to click repeatedly with a pause in between, and exiting after a certain amount of time:

```bash
#!/usr/bin/env bash

cliclick -e 200 m:1002,848 w:200

while true
do
    [[ "$SECONDS" -lt 10 ]] || exit
    cliclick c:1002,848 w:1000 t:Test
done
```

`$SECONDS` is a bash internal variable; the timer stars with the creation of a shell.
